ui-components/__tests__/page_components/LeasingAgent.test.tsx

```diff
@@ -31,4 +31,31 @@ describe("<LeasingAgent>", () => {
       listing.leasingAgentOfficeHours && queryByText(listing.leasingAgentOfficeHours)
     ).toBeNull()
   })
+  it("shows management company details if managementCompany is present", () => {
```

Comment:
These should be two different tests with different titles so that if this test fails it is immediately clear which case it is failing for (thanks for writing tests!)

Reply:
Done, thanks!